src/graph/graph.c

```diff
@@ -182,34 +182,31 @@ static inline size_t _Graph_EdgeCap(const Graph *g) {
 	return g->edges->itemCap;
 }
 
-// Locates edges connecting src to destination.
-void _Graph_GetEdgesConnectingNodes(const Graph *g, NodeID src, NodeID dest, int r, Edge **edges) {
-	ASSERT(g && src < Graph_RequiredMatrixDim(g) && dest < Graph_RequiredMatrixDim(g) &&
-		   r < Graph_RelationTypeCount(g));
-
+static void _CollectEdgesFromEntry
+(
+	const Graph *g,
+	NodeID src,
+	NodeID dest,
+	int r,
+	EdgeID edgeId,
+	Edge **edges
+) {
 	Edge e;
-	EdgeID edgeId;
-	e.relationID = r;
-	e.srcNodeID = src;
-	e.destNodeID = dest;
 
-	// relation map, maps (src, dest, r) to edge IDs.
-	GrB_Matrix relation = Graph_GetRelationMatrix(g, r);
-	GrB_Info res = GrB_Matrix_extractElement_UINT64(&edgeId, relation, src, dest);
-
-	// No entry at [dest, src], src is not connected to dest with relation R.
-	if(res == GrB_NO_VALUE) return;
+	e.relationID  =  r;
+	e.srcNodeID   =  src;
+	e.destNodeID  =  dest;
 
 	if(SINGLE_EDGE(edgeId)) {
-		// Discard most significate bit.
-		edgeId = SINGLE_EDGE_ID(edgeId);
-		e.entity = DataBlock_GetItem(g->edges, edgeId);
-		e.id = edgeId;
+		// discard most significate bit
+		edgeId    =  SINGLE_EDGE_ID(edgeId);
```

Comment:
We change this MSB behavior some 2 days ago

Reply:
we did, but it wasn't merged to `master` that work was done on the `delta` branch, eventually it will be merged.